extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/AutoConfiguredOpenTelemetrySdkBuilderCustomizer.java

```diff
@@ -147,6 +154,16 @@ public Sampler apply(Sampler existingSampler, ConfigProperties configProperties)
         }
     }
 
+    private static Function<String, String> addSlashIfNecessary() {
+        return item -> {
```

Comment:
Can you please convert to an anonymous class? We try to reduce the number of lambdas at runtime.